src/package.xml

```diff
@@ -1,125 +1,50 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<package name="helloworldUncertainty" displayName="Example demonstrating how to use iterations" version="1.1.0" url="https://apexrms.github.io/helloworldEnhanced/">	
-	<transformers>
+<package name="helloworldUncertainty" displayName="Hello World Uncertainty" description="Example demonstrating how to use iterations" version="2.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
+
+	<!--Scenario Datasheets-->
+
+	<!--Run Control-->
+	<dataSheet name="RunControl" displayName="Run Control" isSingleRow="True">
+		<column name="MinimumIteration" displayName="Minimum Iteration" dataType="Integer" defaultValue="1" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="1" isVisible="False"/>
+		<column name="MaximumIteration" displayName="Number of Iterations" dataType="Integer" defaultValue="5" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="1"/>
+		<column name="MinimumTimestep" displayName="Minimum Timestep" dataType="Integer" defaultValue="0" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="0"/>
+		<column name="MaximumTimestep" displayName="Maximum Timestep" dataType="Integer" defaultValue="10" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="0"/>
+	</dataSheet>
+
+	<!--Model Inputs-->
+	<dataSheet name="InputDatasheet" displayName="Inputs" isSingleRow="True">
+		<column name="mMean" dataType="Double" displayName="Slope Distribution Mean"/>
+		<column name="mSD" dataType="Double" displayName="Slope Distribution Standard Deviation"/>
+		<column name="b" dataType="Integer" displayName="Value for b"/>
+	</dataSheet>
+
+	<!--Model Output-->
+	<dataSheet name="OutputDatasheet" displayName="Outputs">
+		<column name="Iteration" dataType="Integer"/>
```

Comment:
Add the iteration and timestep columns to this datasheet this way:

`<dataSheet name="OutputDatasheet" hasIteration="True" hasTimestep="True">`